parlai/core/params.py

```diff
@@ -971,11 +971,29 @@ def parse_known_args(self, args=None, namespace=None, nohelp=False):
         if args is None:
             # args default to the system args
             args = _sys.argv[1:]
+
         args = fix_underscores(args)
+        # handle the single dash stuff. See _handle_single_dash_addarg for info
+        actions = set()
+        for action in self._actions:
+            actions = actions | set(action.option_strings)
```

Comment:
Why not just `add(action)`?

Reply:
I could use `set.update(action.option_strings)` but not `set.add`. option_strings is a list

Reply:
Quick timer check shows that's much faster, so I'll update

Reply:
Ok, I didn't know it is list. It makes sense like this then. Thanks for clarifying.